README.md

```diff
@@ -11,3 +11,40 @@ Publish a preset file for jest. Jest [26](https://jestjs.io/docs/26.x/configurat
 ```
 
 Config was extracted from [lifeomic/typescript-tools](https://github.com/lifeomic/typescript-tools/blob/e38d833a27a04122856fbd5c939f2e26c9bd02d5/config/jest.js)
+
+## expect enhancement
+Matchers can be exposed at the root level.  The library needs to be imported at some level to work.
+
+
+```json
+{
+  "setupFiles": ["@lifeomic/jest-config"]
+}
+```
+
+OR 
+
+```json
+{
+  "setupFilesAfterEnv": ["@lifeomic/jest-config"]
+}
+```
```

Comment:
I found this worked when running `jest` directly, but caused other build steps to fail, e.g., we have a call to `tsc --noEmit` to verify compilation and `tsc` doesn't read `jest.config.js`

Reply:
I followed the instructions from https://github.com/jest-community/jest-extended#setup to try to get this to work.  It's kind of a nuissance.

Reply:
I think the thing that finally gets it is including something in the `include` section of `tsconfig.json`